tools/defaults.bzl

```diff
@@ -38,31 +38,40 @@ def ts_library(name, testonly = False, deps = [], srcs = [], **kwargs):
         **kwargs
     )
 
-def esbuild(
-        name,
-        entry_point,
-        minify = True,
-        sourcemap = "inline",
-        save_to_repo = False,
-        external = [],
-        deps = []):
+def esbuild(platform = "node", target = "node12", **kwargs):
```

Comment:
Is there a case where we don't want to  always target the same version?

Reply:
great point. let's always use this for now. I was thinking there might be cases in the future where we expose browser-specific bundles but that's something we can change once it's the case.